pkg/coveraggregator/simple_file_aggregator.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2024 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package coveraggregator
+
+func MakeSimpleFileLineCoverAggregator(
```

Comment:
As far as I understand the package public interface is only the AggregateStreamData function, if so this shouldn't be exported.